gooddata-sdk/tests/catalog/test_catalog_user_service.py

```diff
@@ -133,14 +138,17 @@ def test_update_user_group(test_config):
     user_group_parent_ids = []
 
     try:
-        sdk.catalog_user.create_or_update_user_group(user_group_id, user_group_parent_ids)
+        user_group_e = CatalogUserGroup.create_user_group(
+            user_group_id=user_group_id, user_group_parent_ids=user_group_parent_ids
+        )
+        sdk.catalog_user.create_or_update_user_group(user_group_e)
         updated_user_group = sdk.catalog_user.get_user_group(user_group_id)
         assert user_group.id == updated_user_group.id
         assert len(updated_user_group.get_parents) == len(user_group_parent_ids)
         assert set(updated_user_group.get_parents) == set(user_group_parent_ids)
     finally:
         sdk.catalog_user.delete_user(user_group_id)
```

Comment:
delete_user on user_group_id seems weird.
Interesting that the test is working, that the final number of user groups is 4.
Check it, please.

Reply:
Checked. Fixed.